engine/crates/engine/src/registry/mod.rs

```diff
@@ -1428,6 +1439,8 @@ pub struct Registry {
     pub operation_limits: OperationLimits,
     #[serde(default)]
     pub trusted_documents: Option<TrustedDocuments>,
+    #[serde(default)]
+    pub cors_config: Option<CorsConfig>,
```

Comment:
Are we planning on loading the whole registry to service CORS requests?  It can be moderately slow to deser if it's large (which I imagine the CC one is), worth bearing in mind

Reply:
It kind of is deserialized when we load the config in every request. We have the config there already, so why not?

Reply:
here https://github.com/grafbase/api/blob/main/workers/gateway/src/lib.rs#L52

Reply:
the registry isn't sent to the gateway. We might be able to store it somewhere else don't remember, but it's the most convenient that comes to mind to get the CORS config there https://github.com/grafbase/api/blob/main/lambda/deployment/gateway-deploy/src/gateway_config.rs#L114
so we can generate the `CustomerDeploymentConfig` with it

Reply:
the registry for v1 is loaded in the executor: https://github.com/grafbase/api/blob/main/workers/executor/src/api.rs#L66